test/configure-jenkins-environment.sh

```diff
@@ -20,8 +20,8 @@ function test_versions() {
     echo >&2 'toVersion and fromVersion selected are equal.. Aborting'
     exit 1
   }
-  to="$(echo "$toVersionShortName" | sed s/-gke//g | tr -d '.')"
-  from="$(echo "$fromVersionShortName" | sed s/-gke//g | tr -d '.')"
+  #to="$(echo "$toVersionShortName" | sed s/-gke//g | tr -d '.')"
```

Comment:
This still needs to be fixed :(  Can you remove the commented code, or what is the best thing to do?

Reply:
Removed as they are unused.